From 920e759ea4f48ca9c8b4dba6dfe5c88d27033121 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <azanella@linux.vnet.ibm.com>
Date: Mon, 17 Jun 2013 15:50:53 -0500
Subject: [PATCH 36/42] PowerPC: Reserve TCB space for EBB framework

This patch reserves four pointer to be used in future Event-Based
Branch framework for PowerPC.
(cherry picked from commit e55a9b256d53c7fc5145e3e4d338d3741b23e232)
---
 nptl/sysdeps/powerpc/tls.h | 5 +++++
 2 files changed, 10 insertions(+)

diff --git glibc-2.17-c758a686/nptl/sysdeps/powerpc/tls.h glibc-2.17-c758a686/nptl/sysdeps/powerpc/tls.h
index 4c09eec..611c773 100644
--- glibc-2.17-c758a686/nptl/sysdeps/powerpc/tls.h
+++ glibc-2.17-c758a686/nptl/sysdeps/powerpc/tls.h
@@ -61,6 +61,11 @@ typedef union dtv
    are private.  */
 typedef struct
 {
+  /* Reservation for the Event-Based Branching ABI.  */
+  uintptr_t ebb_handler;
+  uintptr_t ebb_ctx_pointer;
+  uintptr_t ebb_reserved1;
+  uintptr_t ebb_reserved2;
   uintptr_t pointer_guard;
   uintptr_t stack_guard;
   dtv_t *dtv;
-- 
1.7.11.7

